docs/past_newsletters.md

```diff
@@ -0,0 +1,84 @@
+# Past Newsletter Archive (Akashic Records of Articles)
+
+## __**2023-05-1**__
+- Anime News Scan [Gargi Raj] 
+- Upcoming Action Anime Review [Mayank Rana]
+- Anime Review: Demon Slayer [Nishkal Prakash]
+- Anime Review: I'm Standing on a Million Lives [Aman Tater]
+- Popular Meme Dialogues over the years: Action Anime [Vidunram A R]
+
+## __**2023-04-1**__
+- Anime Review: My Clueless First Friend [Nikhith Manoj]
+- Anime Review: Daily Life of High School Boys [Nishkal Prakash]
+- Anime Review: The Misfit of the Demon King Academy [Maitra Das]
+- Anime Review: School Babysitters [Aman Tater]
+
+## __**2023-03-1**__
+- Anime Review: Cautious Hero [Nishkal Prakash] 
+- Manga Review: In Another World with my Smartphone [Nikhith Manoj]
+- Anime Review: Eminence in Shadow [Phantom Shadow]
```

Comment:
Did you seriously forget Phantom...

Reply:
left it that way coz everyone knows phantom

Reply:
I see...